### PR TITLE
Fixing Issue #56 of duplicated pixels

### PR DESCRIPTION
**Pull request by jorgemarpa**
This PR fixes the duplicated pixel bug.
Here some examples of new behaviero:

image
image

Note: 
- I had to duplicate the masking steps to do nan & saturated pixel masking first, and then mask duplicated pixels.
- Pytest had to be updated to match the correct number of pixels. 
- I also included an extra test to check no duplicated pixels remain and that all valid pixels are kept only once after preprocess 

This solves #56

### CODE REVIEW

**Comment by jorgemarpa** [comment]
I included a couple of lines to add 3 columns before and after bleed columns. We can quickly change this number if we want later.

File: src/psfmachine/tpf.py [hunk 1]
```diff
@@ -1088,10 +1088,11 @@ def _parse_TPFs(tpfs, renormalize_tpf_bkg=True, **kwargs):
     sat_mask = []
     for tpf in tpfs:
         # Keplerish saturation limit
-        saturated = np.nanmax(tpf.flux, axis=0).T.value > 1.4e5
-        saturated = np.hstack(
-            (np.gradient(saturated.astype(float))[1] != 0) | saturated
-        )
+        saturated = np.nanmax(tpf.flux, axis=0).T.value > 1.2e5
```
Comment (christinahedges) [inline src/psfmachine/tpf.py:1091]: This is hardcoded. I think when we switch this up for TESS it'll make things a pain. Let's leave it for now and make sure when we refactor we catch these

File: src/psfmachine/tpf.py [hunk 2]
```diff
@@ -1088,10 +1088,11 @@ def _parse_TPFs(tpfs, renormalize_tpf_bkg=True, **kwargs):
...
+        # add 3 pixels before and after bleed column to be generous
+        for k in range(3):
```
Comment (christinahedges) [inline src/psfmachine/tpf.py:1093]: NICE

**Review by christinahedges — APPROVED** [review]
I think this looks good. My one comment is we could have a test of this saturated stuff by adding a saturated target to our test data. This might be hard to do, or might be adding a significant amount of data to our test data?

**Pull request merged**